scripts/mod_loader/modapi/mechedit.lua

```diff
@@ -0,0 +1,509 @@
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditMoveIconHovered()
+end
+
+function mechedit.IsInventoryHovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditInventoryHovered()
+end
+
+function mechedit.IsInventoryUpHovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditInventoryUpHovered()
+end
+
+function mechedit.IsInventoryDownHovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditInventoryDownHovered()
+end
+
+function mechedit.IsSkill1Hovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditSkill1Hovered()
+end
+
+function mechedit.IsSkill2Hovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditSkill2Hovered()
+end
+
+function mechedit.IsAbilityHovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditAbilityHovered()
+end
+
+function mechedit.SetTestHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetTestHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditTestHighlightable(highlightable)
+end
+
+function mechedit.SetClassHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetClassHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditClassHighlightable(highlightable)
+end
+
+function mechedit.SetHealthIconHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetHealthIconHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditHealthIconHighlightable(highlightable)
+end
+
+function mechedit.SetMoveHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetMoveHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditMoveHighlightable(highlightable)
+end
+
+function mechedit.SetMoveIconHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetMoveIconHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditMoveIconHighlightable(highlightable)
+end
+
+function mechedit.SetNameHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetNameHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditNameHighlightable(highlightable)
+end
+
+function mechedit.SetPilotNameHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetPilotNameHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditPilotNameHighlightable(highlightable)
+end
```

Comment:
⚠ Doesn't appear to be working in my version of the game - the function completes, but the pilot's name is still highlightable / editable

Reply:
I have updated the documentation with `Set..Highlightable` functions with specifics on which functions need to be set each frame, and which are fire and forget functions.
In this case, `MechEdit.SetPilotNameHighlightable` must be called each frame the vanilla behavior should be overridden.